functions/__ssh_agent_is_started.fish

```diff
@@ -7,6 +7,8 @@ function __ssh_agent_is_started -d "check if ssh agent is already started"
 		return 1
 	end
 
-	ps -ef | grep $SSH_AGENT_PID | grep -v grep | grep -q ssh-agent
-	return $status
+	ssh-add -l > /dev/null
```

Comment:
We should add `2>&1` at the end, otherwise it will show the error when it fails.